vehicle/OVMS.V3/components/vehicle_nissanleaf/src/vehicle_nissanleaf.cpp

```diff
@@ -469,6 +469,16 @@ void OvmsVehicleNissanLeaf::IncomingFrameCan1(CAN_frame_t* p_frame)
 
   switch (p_frame->MsgID)
     {
+    case 0x1da:
+      {
+      // Signed value, negative for reverse
+      int16_t nl_rpm = (int16_t)( d[4] << 8 | d[5] );
+      if (nl_rpm != (int16_t)0xffff)
```

Comment:
The 'undefined' value I see is 0x3fff (not 0xffff, which is a valid, if unlikely, value of -1).

Reply:
This change was a result of a  commit done AFTER I created the pull request. Had not expected it to be then automatically included in the existing pull request. Have reverted this change and will create a new pull request for it (using 0x3ffff) once this one is merged.